app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
+    pass
+
+
+class VaccineError(Exception):
+    pass
```

Comment:
You don't need to inherit directly from BaseException. It is not recommended for creating custom exceptions unless you have a specific reason to do so. Use Exception in both classes.